src/Codeception/Test/Gherkin.php

```diff
@@ -91,6 +91,9 @@ public function test()
     protected function validateStep(StepNode $stepNode)
     {
         $stepText = $stepNode->getText();
+        if (\Codeception\Lib\Generator\GherkinSnippets::stepHasPyStringArgument($stepNode)) {
```

Comment:
It would be better to add `use \Codeception\Lib\Generator\GherkinSnippets;` line at the top.